### DISCUSSION TITLE
Guidance

### DISCUSSION THREAD

**Discussion by nasrullahd** [nehal2210/Blockchain-Engineer-Course#4] [Q&A]
Aslam Alikum sir first time GitHub Koo use keyaa haii too contribute nahii hoo paa rha haii woo kese hooga sir?

**Comment by nehal2210** [comment]
is ko thora bht dekh len taake ap ki basic strong ho jaae
https://www.youtube.com/watch?v=Wl61QkGBCmQ

then is ko step by step follow kren ap ko contibute krna aajaae ga
https://firstcontributions.github.io/
